server/server_version.cc

```diff
@@ -139,7 +139,8 @@ const ServerVersion& server_version()
 
 //static const ServerVersion sv(3,1,0);  // JSON-197: Group Encryption. 3.1 branch of Engine & JSON Adapter
 //static const ServerVersion sv(3,22,4); // (forthcoming) P4TB-414: new decrypt_message arg, new message field rating
-static const ServerVersion sv(3,22,5);   // cherry-picking JSON changes from Release_3.1 before retiring the branch
+//static const ServerVersion sv(3,22,5);   // cherry-picking JSON changes from Release_3.1 before retiring the branch
```

Comment:
Why comment this instead of just removing it?

Reply:
To keep the already established history as is.